app/styles/nypr-member-center.scss

```diff
@@ -0,0 +1,98 @@
+@import "compass/css3/flexbox";
+
+@import "ember-modal-dialog/ember-modal-structure";
+@import "ember-modal-dialog/ember-modal-appearance";
+
+@import "nypr-ui/vars";
+@import "nypr-ui/buttons";
```

Comment:
Notes for myself: this file should be converted to partial, and double imports removed.